pida-clients/oauth-client/src/main/kotlin/com/pida/client/oauth/OAuthService.kt

```
@@ -14,11 +14,13 @@ class OAuthService(
         try {
             kaKaoClient.getUserInfo(token)
         } catch (e: FeignException) {
-            if (e.status() == 401) {
+            if (e.status() in setOf(400, 401, 403)) {
                 throw AuthenticationErrorException(AuthenticationErrorType.INVALID_KAKAO_TOKEN)
             } else {
-                throw AuthenticationErrorException(AuthenticationErrorType.INVALID_KAKAO_TOKEN, e.message)
+                throw AuthenticationErrorException(AuthenticationErrorType.KAKAO_AUTH_PROVIDER_UNAVAILABLE, e.message)
             }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)